### PR TITLE
Clean up hyperflux loose ends

### PR DESCRIPTION
**Pull request by HexaField**
## Summary

- [x] Remove $from field from actions
- [x] Remove UserID from hyperflux
- [x] Move PeerID to hyperflux

## References
closes #_insert number here_

### CODE REVIEW

File: packages/client-core/src/admin/services/SceneService.ts [hunk 1]
```diff
@@ -40,7 +41,7 @@ export const AdminSceneState = defineState({
     fetched: false,
     updateNeeded: true,
     lastFetched: Date.now(),
-    singleScene: { scene: {} } as SceneDataType
+    singleScene: { scene: {} } as Omit<SceneDataType, 'scene'> & { scene: SceneJsonType }
```
Comment (DanielBelmes) [inline packages/client-core/src/admin/services/SceneService.ts:44]: the type gymnastics here are a bit funky. Is typebox not getting the right type from the schema?

Comment (HexaField) [reply packages/client-core/src/admin/services/SceneService.ts:44]: No, the type was dependent on the ecs package, which common can not have. This stuff will be replaced with gltf soon, so it's more or less temporary.

Comment (DanielBelmes) [reply packages/client-core/src/admin/services/SceneService.ts:44]: oh scrolled down. Sorry. Was working off local from when I clicked in github. I guess it changes the question to. You can't pass SceneJsonType to typebox?

File: packages/projects/ProjectConfigInterface.ts [hunk 2]
```diff
@@ -65,7 +65,7 @@ export interface ProjectConfigInterface {
    * This is loaded on ALL instances of the engine if enabled
    * Allows for running of custom logic regardless of which scene or route is loaded
    */
-  worldInjection?: () => Promise<{ default?: () => Promise<void> }>
+  worldInjection?: () => Promise<{ default?: () => Promise<void> } | any>
```
Comment (DanielBelmes) [inline packages/projects/ProjectConfigInterface.ts:68]: was this meant to stay? I feel like we can have a stricter type here

File: packages/client-core/src/admin/services/SceneService.ts (hunk 1 above)
Comment (HexaField) [reply packages/client-core/src/admin/services/SceneService.ts:44]: We might be able to

**Review by DanielBelmes — APPROVED** [review]
Alright looks good. Though please double check the world injection type.

**Pull request merged**